Makefile.am

```diff
@@ -83,13 +86,14 @@ tester_blargg_SOURCES = $(TESTER_SOURCES)
 
 AM_CPPFLAGS = -I$(top_srcdir)/include
 
-vpnes_CPPFLAGS = $(AM_CPPFLAGS) $(SDL_CFLAGS)
-vpnes_LDADD = $(SDL_LIBS) libcore.a
+vpnes_CPPFLAGS = $(AM_CPPFLAGS) $(SDL_CFLAGS) $(BOOST_CPPFLAGS) $(BOOST_PROGRAM_OPTIONS_CPPFLAGS)
+vpnes_LDADD = $(SDL_LIBS) $(BOOST_LDFLAGS) $(BOOST_PROGRAM_OPTIONS_LIB) libcore.a
 
 unittests_CPPFLAGS = $(AM_CPPFLAGS) $(BOOST_CPPFLAGS) $(BOOST_UNIT_TEST_FRAMEWORK_CPPFLAGS)
 unittests_LDADD = $(BOOST_LDFLAGS) $(BOOST_UNIT_TEST_FRAMEWORK_LIB) libcore.a
 
-tester_blargg_LDADD = libcore.a
+tester_blargg_CPPFLAGS = $(AM_CPPFLAGS) $(SDL_CFLAGS) $(BOOST_CPPFLAGS) $(BOOST_PROGRAM_OPTIONS_CPPFLAGS)
+tester_blargg_LDADD = $(SDL_LIBS) $(BOOST_LDFLAGS) $(BOOST_PROGRAM_OPTIONS_LIB) libcore.a
```

Comment:
Same as above, `libcore.a` depends on `-lboost_program_options`, so the order is wrong